src/store/src/Bridge/Postgres/PostgresHybridStore.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Bridge\Postgres;
+
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Platform\Vector\VectorInterface;
+use Symfony\AI\Store\Document\Metadata;
+use Symfony\AI\Store\Document\VectorDocument;
+use Symfony\AI\Store\Exception\InvalidArgumentException;
+use Symfony\AI\Store\ManagedStoreInterface;
+use Symfony\AI\Store\StoreInterface;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * Hybrid Search Store for PostgreSQL/Supabase
+ * Combines pgvector (semantic) + PostgreSQL Full-Text Search (ts_rank_cd) using RRF.
+ *
+ * Uses Reciprocal Rank Fusion (RRF) to combine vector similarity and full-text search,
+ * following the same approach as Supabase hybrid search implementation.
+ *
+ * Requirements:
+ * - PostgreSQL with pgvector extension
+ * - A 'content' text field for full-text search
+ *
+ * @see https://supabase.com/docs/guides/ai/hybrid-search
+ *
+ * @author Ahmed EBEN HASSINE <ahmedbhs123@æmail.com>
+ */
+final readonly class PostgresHybridStore implements ManagedStoreInterface, StoreInterface
```

Comment:
let's just call it `HybridStore` instead
```suggestion
final readonly class HybridStore implements ManagedStoreInterface, StoreInterface
```

Reply:
@chr-hertel Thanks for reviewing. I've just renamed it.